otter/convergence/model.py

```diff
@@ -563,11 +579,22 @@ def equivalent_definition(self, other_description):
                 other_description.port == self.port)
 
 
+@attr.s
+class CLB(object):
+    """
+    Cloud load balancer
+    """
+    # Is health monitor enabled?
+    health_monitor = attr.ib(default=False)
+
+
 @implementer(ILBNode, IDrainable)
 @attributes([Attribute("node_id", instance_of=basestring),
              Attribute("description", instance_of=CLBDescription),
              Attribute("address", instance_of=basestring),
              Attribute("drained_at", default_value=0.0, instance_of=float),
+             Attribute("status", instance_of=NamedConstant,
```

Comment:
Can't tell about the future but the [documentation](https://developer.rackspace.com/docs/cloud-load-balancers/v1/api-reference/nodes/) only mentions `OFFLINE` and `ONLINE` as of now. I think it is a good idea to have boolean attribute `online` except that future reader will need to know that it is correlated with `status` field.